public/js/index.js

```diff
@@ -24,7 +25,23 @@ $("#loginForm").on("submit", function(e) {
   //if login passes, we will serialize the bug data and run the metrics function (passing the bug object as an argument)
```

Comment:
Is the plan to store the log in information in a .env, since it's single user? Or something like that?

Reply:
I'm not actually sure. I'm fine with whatever ends up being the plan of action with the backend, whether it's proper authentication or just... I dunno. I know very little about how that'll end up being done.